libraries/chain/include/eosio/chain/kv_context.hpp

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <fc/log/logger.hpp>
```

Comment:
This include does not appear to be necessary.

Reply:
Yeah it’s not anymore indeed with the changes done on the review.

Did you fix it after the merge or I open a fix PR?

Reply:
It's not a big deal. We can remove it later.